test/components/inputs/inputvalidator.coffee

```diff
@@ -0,0 +1,109 @@
+should = require 'should'
+sinon = require 'sinon'
+shouldSinon = require 'should-sinon'
+KDInputValidator = require '../../../lib/components/inputs/inputvalidator'
+
+describe 'KDInputValidator', ->
+  beforeEach ->
+    @sinon = sinon.sandbox.create()
+    @instance = new KDInputValidator
+
+  afterEach ->
+    @sinon.restore()
+
+  it 'exists', ->
+    KDInputValidator.should.exist
+
+  describe 'ruleRequired', ->
+    it 'should validate', ->
+      input =
+        getValue: ->
+          ''
+        getOptions: ->
+          { validate: yes }
```

Comment:
it's better if we test these with a real input instance, if we change the `InputView` these tests will continue working, even though they're tightly coupled.

just instantiate an input during `beforeEach` and instead of mocking getters here do `@input.setValue` and `@input.setOptions`